tiledb/sm/filesystem/posix.h

```diff
@@ -292,19 +288,8 @@ class Posix {
   static Status write_at(
       int fd, uint64_t file_offset, const void* buffer, uint64_t buffer_size);
 
-  /**
-   * Parse config to get posix permissions for creating new files
-   * @param permissions parsed permissions are set to this parameter
-   * @return Status
-   */
-  Status get_posix_file_permissions(uint32_t* permissions) const;
-
-  /**
-   * Parse config to get posix permissions for creating new directories
-   * @param permissions parsed permissions are set to this parameter
-   * @return Status
-   */
-  Status get_posix_directory_permissions(uint32_t* permissions) const;
+ private:
+  uint32_t file_permissions_, directory_permissions_;
```

Comment:
This should probably end up in a `class PosixFSParams`, analogous to `VFSParams`. This is the same spirit, and is small enough to not worry about it now.